### ISSUE TITLE
Download to upload ratio should count against actual download stats

### ISSUE THREAD

**Issue by kode54** [transmission-remote-gtk/transmission-remote-gtk#90]
Currently, the ratio counts the total size of the torrent toward the ratio when the torrent has completed, rather than counting the actual bytes downloaded. This results in some funny stats, say, if you "upgraded" to a new version of a file set torrent, and verified over 95% of the data to be present against the new .torrent file, and download only the difference.

**Comment by u2n** [comment]
Agree, the ratio should be calculated from torrent download start, not end. Sometimes a high ratio is reached even before the download halfway mark (typically occurs when there are many leechers and no seeds). The currently displayed ratio is useless to flag that condition. It could, however, be helpful if modified.
